apps/tenant-management-webapp/src/app/pages/admin/services/pdf/styled-components.ts

```diff
@@ -40,7 +40,7 @@ export const BadgeWrapper = styled.div`
 `;
 
 export const ButtonBox = styled.div`
-  width: 2.15rem;
+  width: 1.75rem;
```

Comment:
I would suggest mapping the width to the most closed css variable.

Reply:
I thought to but we are missing exact alignment with the other controls. So I made change without css variable